src/element.ts

```diff
@@ -2,6 +2,7 @@
 // MIT License
 // @author Mohit Cheppudira
 
+import { BEAM_BOTH } from './beam';
```

Comment:
Is this import used? I don't see it below. Why does element base class need to know about beams?

Reply:
removed